src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,388 @@ tree_item_index: 1100
 
 {{! excerpt}}
 The [Nuxeo Aspera](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-aspera) connector
-enables users to download/upload binaries with Aspera.
+enables users to download/upload binaries with Aspera.  It is designed to be used with **Aspera On Cloud** instances, however, if you use a **self-hosted Aspera** instance, it is still possible you will be able to use the Nuxeo Aspera Connector.  Contact your Aspera team for more information and assistance.
 {{! /excerpt}}
 
-## Requirements
+## Version 3.0.0
+### Requirements
+
+- Aspera Desktop Client - you will be prompted to install necessary versions when using the Connector.
+- Nuxeo Server LTS 2019 (**10.10, HF 31, at least**) with access to AWS S3 Storage along with the [Amazon S3 Online Storage plugin](https://connect.nuxeo.com/nuxeo/site/marketplace/package/amazon-s3-online-storage?version=11.2.13)
+- IBM Aspera on Cloud Subscription (see note above about Aspera self-hosted)
+
+
+{{#> callout type='warning'}}
+If you are **upgrading from a previous version** of the Nuxeo Aspera connector, please see the [upgrade notes](https://doc.nuxeo.com/nxdoc/nuxeo-aspera-connector/nuxeo-aspera-upgrade-notes/) for critical information related to potential breaking code.
+{{/callout}}
+
+### Installation
+
+Installation is made of two steps:
+
+1.  Install the [Nuxeo Package](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-aspera) available from the marketplace.
+2.  Install the Aspera desktop client (you will be prompted to install the necessary Aspera software when you try to use the Connector, if you do not already have it).
+
+### Configuration
+
+#### Aspera Configuration
+
+We need to configure 2 Aspera nodes; one for upload and one for download.
+
+Each node will be attached to one S3 bucket in Nuxeo:
+- The main Nuxeo S3 bucket in Nuxeo for download purpose
```

Comment:
@lisamcintyre , since it is related we should make sure about the download flow as discussed on slack. (i.e. Download will work without S3 Direct Upload?)

Reply:
@nmpcunha tested a download on master where the files were uploaded just using webui.  It worked, so yes, we should be ok.